Modules/@babylonjs/react-native-windows/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -168,4 +170,22 @@ namespace winrt::BabylonReactNative::implementation {
     {
         BabylonNative::RenderView();
     }
+
+    void EngineView::UpdateProperties(IJSValueReader const& reader)
+    {
+        auto const& propertyMap = JSValueObject::ReadFrom(reader);
+
+        for (auto const& pair : propertyMap)
+        {
+            auto const& propertyName = pair.first;
+            auto const& propertyValue = pair.second;
+
+            if (propertyName == "isTransparent")

```

Comment:
I admit I'm not 100% sure how the native properties propagate in React Native, but is there a possibility of clearing `isTransparent` from the map at runtime? Do we need to also handle the case where `isTransparent` goes from being true to just unset? Right now that'd currently result in the view remaining transparent.

Reply:
Same as iOS/Android version, transparency doesn't change until property is changed.